src/Compilers/CSharp/Portable/Lowering/DiagnosticsPass_ExpressionTrees.cs

```diff
@@ -658,7 +671,7 @@ private BoundNode VisitMethodGroup(BoundMethodGroup node, bool parentIsConversio
             }
 
             CheckReceiverIfField(node.ReceiverOpt);
-            return base.VisitMethodGroup(node);
```

Comment:
>return base.VisitMethodGroup(node); [](start = 12, length = 35)

Why is this a right thing to do? Couldn't there be scenarios when ```VisitMethodGroup``` is called implicitly and, therefore, the extra checks added in VisitConversion wouldn't be performed?

Reply:
@AlekseyTs The problem is that the extra checks require knowing which method was chosen by overload resolution and `BoundMethodGroup` does not contain that information. I believe `VisitMethodGroup` can now be called implicitly only for invalid code (e.g. when the parent is `BoundBadExpression`) and so it shouldn't matter that the extra checks are skipped.